aiidalab_widgets_base/structures.py

```diff
@@ -136,49 +123,68 @@ def store_structure(self, label=None, description=None):
         self.structure_node.store()
         print("Stored in AiiDA: " + repr(self.structure_node))
```

Comment:
I thin it's a bad idea to report to the user using print-statements as part of the widget implementation.

Reply:
agreed. Put the output into `ipw.HTML`.